src/main/scala/eu/proteus/job/kernel/ProteusJob.scala

```diff
@@ -127,21 +151,45 @@ object ProteusJob {
         loadConsumerKafkaProperties))
 
     // simple moments
+    if(ProteusJob.LinkMoments){
+      LOG.info("Moments output topic: simple-moments")
+      val moments = MomentsOperation.runSimpleMomentsAnalytics(source, 54)
+      implicit val momentsTypeInfo = createTypeInformation[MomentsResult]
+      val momentsSinkSchema = new UntaggedObjectSerializationSchema[MomentsResult](env.getConfig)
 
-    val moments = MomentsOperation.runSimpleMomentsAnalytics(source, 54)
-    implicit val momentsTypeInfo = createTypeInformation[MomentsResult]
-    val momentsSinkSchema = new UntaggedObjectSerializationSchema[MomentsResult](env.getConfig)
-
-    //val momentsSinkSchema = new SimpleStringSchema()
-
-    val producerCfg = FlinkKafkaProducer010.writeToKafkaWithTimestamps(
+      val producerCfg = FlinkKafkaProducer010.writeToKafkaWithTimestamps(
         moments.javaStream,
         "simple-moments",
         momentsSinkSchema,
         loadBaseKafkaProperties)
 
-    producerCfg.setLogFailuresOnly(false)
-    producerCfg.setFlushOnCheckpoint(true)
+      producerCfg.setLogFailuresOnly(false)
+      producerCfg.setFlushOnCheckpoint(true)
+    }
+
+    if(ProteusJob.LinkSAX){
+
+      val variables = parameters.get("sax-variable").split(",")
```

Comment:
let's use getRequired here instead of get. This will ensure a safe-failing in case the parameter is not set.